core/internal-groups.rst

```diff
@@ -0,0 +1,52 @@
+Internal Groups and Areas of Focus
+---------------------------------
+
+Core is probably the big component in coala. Almost everything else is heavily
+tied with it; so it doesn't make sense to have a huge, monolithic, and
+unfocussed team. So I propose to have a few areas of focus that each team member
+can choose to align themselves with.
+
+Please note that these are not strict teams. And by no means exhaustive. These
+is simply a categorization to co-ordinate efforts better. And it is entirely
+possible than an issue doesn't fall under one of the proposed groups squarely -
+overlaps are allowed. In short, these is simply a way to organize better, in an
+undivisive manner.
+
+Performance
+===========
+
+I'm a sucker for performance and I like things fast. And you should too!
+
+This group intends to improve on coala's performance. Since coala is written in
+Python, there are some limitations to how far we can go with this, but there's
+still quite a lot we can do.
+
+Usability and User Interface
+============================
+
+coala is designed to be very user-friendly and we should work to keep it that
+way. This means constant feedback, design and implementation. This also involves
+coming with new and intuitive interfaces. Just like fashion, UI development
+never ends.
+
+`bearlib` Development
+========================
+
+coala is nothing without its bears. This group will be dedicated to extending,
+improving, and maintaining the library available to bear authors.
+
+Supporting Tools
```

Comment:
Initially, I wasn't entirely for a separate team for each tool. But the `cib` team exists, while a `coala-quickstart` or `coala-html` team doesn't exist. So somebody has to be responsible for them and the core team's description ("anything that isn't covered by another team falls under the core team").

So to answer your question, `cib` is its own team, with its own roadmap and team members. But of course, since it has a lot of dependence on `coalib`, there'll be overlap. However, `coala-quickstart` is a subset of the core team IMO.